ui.tests/test-module/specs/fileinput/fileinput.runtime.spec.js

```diff
@@ -107,6 +107,19 @@ describe("Form with File Input - Basic Tests", () => {
                 checkHTML(id, field.getModel().getState())
         });
     })
+    it(" after attaching the file widget value is reset to allow duplicate file ", () => {
+        const sampleFileName = 'sample2.txt', fileInput = "input[name='fileinput1']";
+        cy.get(fileInput).should('have.value', "");
+        cy.get(fileInput).attachFile(sampleFileName).then(() => {
```

Comment:
Can you improve the test case to incorporate the attach of duplicate file use-case ? Right now you are just checking widget value.